index.html

```diff
@@ -25,5 +25,107 @@ <h1 class="headline-title">I'm Dhiraj<br>Glad to see<br>you!</h1>
             <li><a href="#"><img src="images/angellist.png" alt="Angellist"></a></li>
         </ul>
     </section>
+    <section class="project">
+        <div class="card">
```

Comment:
- [ ] - [ OPTIONAL ] Well done on creating the readme file, but at this step, we agree that it should have all the necessary information about the project, what do you think if you make a few touches by adding the project images, description, title and remove the microverse logo, please

- [ ] Your code is well structured referring to the project requirements, congratulations @dhirajsingha143, but we have a few things to modify before the project is approved, let's start with the design below, your design is well done but we are missing some space as indicate the red marks compared to the original design, please make use you adjust them. 💯 
<img width="678" alt="lang" src="https://user-images.githubusercontent.com/77843050/205432820-9dc7e417-602c-4f2d-bc1c-c51ec708da1d.png">

- [ ] Take a look at this record below.

![lang](https://user-images.githubusercontent.com/77843050/205432921-88ade861-e7a8-479f-9858-e86b99f54cf6.gif)

- [ ] - [ OPTIONAL] I think we see the same thing right? starting with the logo name, you can take it as optional BUT I would suggest you change that text into something relevant for you as your name, in addition to that, the position property you gave it is causing a few problems when we scroll, there are 2ways you can fix this, one is by providing a background to the nav bar with the same exact color as you `section` with class `headline` and the second is to remove the fixed position so when we scroll, the nav bar goes with the whole page, I let you choose one 😄   

- [ ] You have indeed remarked on the blank spaces on the right side of your page, right? your sections are not well-adjusted in the middle of the page, please fix that too